decks/Elections-Proposal.mdx

```diff
@@ -0,0 +1,65 @@
+import { CodeSurfer as Surfer } from "code-surfer";
+import { CodeSurferColumns, Step } from "code-surfer";
+import { Appear, Background } from "gatsby-theme-mdx-deck";
+import * as L from "../src/layout";
+import customTheme from "../src/theme";
+import GreetingLoader from "./src/greeting-loader";
+
+export const theme = customTheme;
+
+# Workshop Proposal
+
+---
+
+# Specific Topic
+- We are going to use the OpenFEC API Election endpoint to visualize data for total disbursements and total receipts for Presidential candidates for the last 30 elections.
+- Graph will compare this financial information over time and for both the Democratic and Republican parties. 
+
+---
+
+# Interactive Component
+- We will have students visualize data from the same endpoint but for cash_on_hand_end_period 
+---
+
+# Code Deliverables
+## Week 2
+- Code out total disbursements and total receipts (Work on together)
+
+## Week 3
+- Finish up any unfinished work from week 2 + complete interactive activity (Work on together)
+
+## Week 4
+- Finish up any unfinished code from interactive activity
+- Work on presentation slides (will decide how to split up)
+---  
```

Comment:
also try to decide the layout of the code end-to-end. i.e. introduce syntax gotchas, simple definitions, function meanings...